packages/generator-one-app-module/generators/app/templates/base-child-module/__tests__/components/ModuleContainer.spec.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import <%=moduleNamePascal%> from '../../src/components/<%=moduleNamePascal%>';
+
+describe('<%=moduleNamePascal%>', () => {
+  it('module should render correct JSX', () => {
```

Comment:
How about `should render as expected`? That way the test is describing the behavior and not the implementation.

Reply:
fixed in all template test files